.gitignore

```diff
@@ -27,3 +27,16 @@ config*.json
 # Pyinstaller files
 build
 dist
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
```

Comment:
These seem quite specific as backup files, will other venv tools create different extensions?  I'm wondering whether we just block `*.bak` or `env.*`...

Reply:
I've researched a lot of cases. 
`.bak` seems to be the type of folder and file name you specify when you want to back up. It's not generated by the program. `env.` I think it's better to take the form. I think this is a little bit more general and it could cover a lot of forms.

Reply:
However, I think the dots behind the `env` can be used to distinguish various deployment stages such as `.bak`, `.prod`, `.dev`, etc. Since volatility3 is a difficult case to use right away, I think ignoring the `env` `venv` folder is an improvement.

Reply:
Yeah, that's fair enough.  Thanks for making the changes  5:)